bot_api_py/events/death_event.py

```diff
@@ -0,0 +1,13 @@
+from bot_api_py.events.bot_event import BotEvent
+
+class DeathEvent(BotEvent):
```

Comment:
This class lacks the `DeathEvent(int turnNumber) { super(turnNumber); }` (__init__ method) as the user should be able to read out the `turn_number`